.github/workflows/docker.yml

```diff
@@ -78,7 +73,7 @@ jobs:
         uses: RDXWorks-actions/cache/restore@main
         with:
           path: /tmp/outputs/cache/docker
-          key: babylon-node-default-${{ hashFiles('./Dockerfile') }}
+          key: babylon-node-default-${{ hashFiles('./Dockerfile.v2') }}
```

Comment:
Now that we're merging things together, can we replace `./Dockerfile` with `./Dockerfile.v2`? :)